<?xml version="1.0"?>
<launch>
    <arg name="base_frame" default="base_link"/>
    <arg name="map_frame" default="odom"/>
    <arg name="pose_topic" default="current_pose"/>

    <node pkg="dummy_localizer" type="dummy_localizer_node" name="dummy_localizer_node" output="screen" respawn="true">
        <param name="base_frame" value="$(arg base_frame)"/>
        <param name="map_frame" value="$(arg map_frame)"/>
        <param name="pose_topic" value="$(arg pose_topic)"/>
    </node>
</launch>
